### PR TITLE
CTOR-724 : Plugin(network::nokia::timos::snmp) : New mode sas-alarm

### PR DESCRIPTION
**Pull request by lucie-dubrunfaut**
## Description

Add mode sas-alarm

## Target version (i.e. version that this PR changes)

- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] 24.04.x
- [ ] 24.10.x
- [ ] 25.10.x
- [ ] Cloud
- [x] Monitoring Connectors

### CODE REVIEW

File: pp/integrations/plugin-packs/procedures/network-nokia-timos-snmp.md [hunk 1]
```diff
@@ -20,24 +26,25 @@ The connector brings the following service templates (sorted by the host templat
 
 | Service Alias | Service Template                     | Service Description                         |
 |:--------------|:-------------------------------------|:--------------------------------------------|
-| Cpu           | Net-Nokia-Timos-Cpu-SNMP-custom      | Check the rate of CPU utilization    |
+| Cpu           | Net-Nokia-Timos-Cpu-SNMP-custom      | Check the rate of the utilization of CPU    |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-nokia-timos-snmp.md:29]:
```suggestion
| Cpu           | Net-Nokia-Timos-Cpu-SNMP-custom      | Check the rate of CPU utilization    |
```

File: pp/integrations/plugin-packs/procedures/network-nokia-timos-snmp.md [hunk 2]
```diff
@@ -20,24 +26,25 @@ The connector brings the following service templates (sorted by the host templat
...
 | Hardware      | Net-Nokia-Timos-Hardware-SNMP-custom | Check hardware                              |
-| Memory        | Net-Nokia-Timos-Memory-SNMP-custom   | Check the rate of memory utilization |
+| Memory        | Net-Nokia-Timos-Memory-SNMP-custom   | Check the rate of the utilization of memory |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-nokia-timos-snmp.md:31]:
```suggestion
| Memory        | Net-Nokia-Timos-Memory-SNMP-custom   | Check the rate of memory utilization |
```

File: pp/integrations/plugin-packs/procedures/network-nokia-timos-snmp.md [hunk 3]
```diff
@@ -313,139 +330,149 @@ yum install centreon-plugin-Network-Nokia-Timos-Snmp
...
+|:---------------|:----------------------------------------------------------------------------------------------------------------------------------------------------------|:---------------------------------------------------------------------------------------------------------------------------------|:-----------:|
+| FILTERNAME     | Filter alarm input (tmnxSasAlarmInputDescription) (can be a regexp)                                                                                       |                                                                                                                                  |             |
+| WARNINGSTATUS  | Set warning threshold for status. You can use the following variables: %{alarm\_input\_admin\_state}, %{alarm\_input\_status}, %{alarm\_output\_severity} | %{alarm\_input\_admin\_state} eq "up" and %{alarm\_input\_status} eq "alarm" and %{alarm\_output\_severity} =~ /minor/           |             |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-nokia-timos-snmp.md:465]: ?

**Review by cg-tw — CHANGES_REQUESTED** [review]

File: pp/integrations/plugin-packs/procedures/network-nokia-timos-snmp.md (hunk 3 above)
Comment (lucie-dubrunfaut) [reply pp/integrations/plugin-packs/procedures/network-nokia-timos-snmp.md:465]: Le help dit : 
```
=item B<--warning-status>

Set warning threshold for status. (Default: '%{alarm_input_admin_state} eq "up" and %{alarm_input_status} eq "alarm" and %{alarm_output_severity} =~ /minor/')
You can use the following variables: %{alarm_input_admin_state}, %{alarm_input_status}, %{alarm_output_severity}
```

Donc ça donne : 
Set warning threshold for status. You can use the following variables: %{alarm_input_admin_state}, %{alarm_input_status}, %{alarm_output_severity}

Je comprends pas l'interrogation ^^'

**Review by cg-tw — APPROVED** [review]

**Pull request merged**